CONTRIBUTING.rst

```diff
@@ -0,0 +1,133 @@
+
+If you have
+suggestions on new features, improvements to the project itself, you've
+detected some unruly behavior or a bug, or a suggestion of how we can
+make the project more accessible, feel free to file an `issue on
+github <https://github.com/data-exp-lab/widgyts/issues)>`_. 
+
+Communication Channels
+----------------------
+
+If you need help or have any questions about using widgyts that's beyond the
+documentation (or if you'd like to join our community), you're welcome to
+join the `yt project's slack <https://yt-project.org/slack.html>`_ (specifically in the widgyts channel) 
+or ask in the `yt users mailing list
+<https://mail.python.org/mailman3/lists/yt-users.python.org/>`_.
+
+If you'd like to talk about new features or development of widgyts, the widyts
+channel in the `yt project's slack <https://yt-project.org/slack.html>`_ is a
+good place to go. The `yt development mailing list 
+<https://mail.python.org/mailman3/lists/yt-dev.python.org/>`_ is channel where
+these discussions are welcomed. 
+
+Contributing Code
+-----------------
+
+We would be delighted for you to join and work on the widgyts project! If
+you're interested in getting started, browse some of our `open issues
+<https://github.com/data-exp-lab/widgyts/issues>`_ and see if there's anything
+that you may find interesting. If there's a new feature you'd like to add that
+isn't in open issues, please reach out in the `widgyts slack channel in the yt
+slack <https://yt-project.org/slack.html>`_ or on the `yt development mailing 
+list <https://mail.python.org/mailman3/lists/yt-dev.python.org/>`_ to talk a 
+bit more about
+what you'd like to contribute. This will help make your contribution review go
+smoothly and merge quickly!
+
+To get a development environment set up on your machine, please see the
+:ref:`development_install` directions to get started. 
+
+When issuing a pull request for new features in the widgyts package, please 
+make sure the following are satisfied:
+
+- new features have accompanying documentation, including docstrings and
+  examples
+- if javascript functionality is added, ensure it is commented thoroughly
+- tests have been added for new features
+- all tests run and pass
+- new documentation satisfies documentation contribution requirements
+
+.. _building_the_documentation:
+
+Contributing Examples or Documentation
+--------------------------------------
+
+To contribute new examples or update the documentation you do not need to have
+a development build of widgyts on your personal machine. To build the
+documentation, we have opted to use the same method as `ipywidgets
+<https://ipywidgets.readthedocs.io/en/stable/dev_docs.html>`_ and distribute an
+``environment.yml`` file that can be used to create an environment with the
+necessary packages to build the documentation on your personal machine. 
+
+To install a widgyts docs environment with conda::
+
+  $ conda env create -f docs/environment.yml
```

Comment:
Interesting.  We might want to note that if you're already developing ipywidgets, just use that dev env.

Reply:
Good point! I'll add a sentence here about that.

(This isn't an environment.yml for ipywidgets development though. It's for the ipywidgets documentation specifically.)